### PR TITLE
address issue #340 re period displayed on timecard entry form

### PR DESCRIPTION
**Pull request by batemapf**
- makes small ux and content changes, including updating 18f logo
- adds new 18f logo to assets
- addresses #340

### CODE REVIEW

File: tock/tock/static/css/style.css [hunk 1]
```diff
@@ -204,10 +204,10 @@ picture {
   max-width: 100%; }
 
 .banner {
```
Comment (brendansudol) [inline tock/tock/static/css/style.css]: this css needs some formatting / linting cleanup in general -- i will do this when i get a chance soon -- but i don't think it should block this PR

**Comment by brendansudol** [comment]
🚢 😄

**Comment by brendansudol** [comment]
not sure why this is red -- it passed on the checks but then i accidentally clicked the "close issue" button instead of "merge" and it went red -- i think we can safely ignore

**Comment by batemapf** [comment]
looks like it went red b/c it was closed before travis ci build completed. concur, safe to ignore :-)

**Pull request merged**